game_view.cpp

```diff
@@ -259,6 +282,18 @@ void game_view::show() noexcept
         draw_shelters();
     }
 
+    // Set fourth view for players coordinates
+    #ifndef NDEBUG  // coordinates should not be visible in release
+    sf::View player_coords_view(
+                sf::Vector2f(m_window.getSize().x / 4.5, m_window.getSize().y / 4.5),
+                sf::Vector2f(m_window.getSize().x / 2, m_window.getSize().y / 2)
+                );
+    player_coords_view.setViewport(sf::FloatRect(0.5f, 0.0f, 0.5f, 0.5f));
+    m_window.setView(player_coords_view);
```

Comment:
You should make this a function `game_view::set_player_coords_view()` and maybe call it inside `draw_player_coords()`